pkg/k8s/policy.go

```diff
@@ -0,0 +1,367 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes"
+)
+
+// ServerAndAuthorization holds a server name and serverauthorization name.
+type ServerAndAuthorization struct {
+	Server              string
+	ServerAuthorization string
+}
+
+type id struct{ name, namespace string }
+
+var sazGVR = schema.GroupVersionResource{
```

Comment:
Looks like we can use these and remove them from https://github.com/linkerd/linkerd2/blob/alex/viz-authz/viz/metrics-api/stat_summary.go#L28